### DISCUSSION TITLE
Folder structure vs network structure vs tags

### DISCUSSION THREAD

**Discussion by quesada** [zim-desktop-wiki/zim-desktop-wiki#1873] [Ideas]
Right now we have two ways to organize info in zim: Folder structure vs network structure.
They are overlapping in functionality, right?
Example:
1. I could create parent note (a folder) called 'business" and have lots on notes under that note
2. you could link a bunch of notes to a note called "business"
3. you could use tags, and then list all notes tagged 'business" for an overview using the tag sidepanel

Of those solutions, only 2 benefits from the 'network visualization' (graphviz), which I think is awesome.

What method do you use?

**Comment by jaap-karssenberg** [comment]
Using the folder structure has the limitation that a page can belong to
only one folder. With links and take you can create many mappings at the
same time.

Tags are a useful addition for structuring in case you also use links as
"normal" links in text.

Indeed there is no visualization for tags, but there is a special index
side pane for tags.

Hope this helps,

Jaap


Op zo 16 jan. 2022 13:02 schreef Jose Quesada ***@***.***>:
